### DISCUSSION TITLE
后台登录不上的原因找到了

### DISCUSSION THREAD

**Discussion by jackloves111** [halo-dev/halo#3250] [General]
### 是什么版本出现了此问题？

2.2.0

### 使用的什么数据库？

H2

### 使用的哪种方式部署？

Docker

### 发生了什么？

后台登录不上找到原因了，因为安装过程中的外部访问地址和实际地址不一致，后台就无法登录，我用域名可以登录，但是切换成内网IP就无法登录了。

  --halo.external-url=http://test.com \

也就是用http://test.com可以访问后台，用http://192.168.1.134:8090/ 无法访问后台，会提示密码不正确。

**Comment by ruibaby** [comment]
因为登录需要同域，所以我在 https://github.com/halo-dev/halo/issues/3228#issuecomment-1416923754 中说看看浏览器日志。
